### DISCUSSION TITLE
Cannot build (from source) Freeplane 1.8.6

### DISCUSSION THREAD

**Discussion by rubisara38** [freeplane/freeplane#984] [Help] [answered]
My environment is Windows 10 x64.

I can build from source successfully Freeplane versions 1.10.3 or later, but I cannot do that with version 1.8.6.

Specifically, when I ran 'gradle build' command, I got several errors like this:

Could not GET 'https://repo.spring.io/plugins-release/com/lightdev/app/shtm/simplyhtml/SimplyHTML/0.17.7/SimplyHTML-0.17.7.pom'. Received status code 401 from server...

Full error log is in this attached text file:
[FP_1_8_6_build_error_log.txt](https://github.com/freeplane/freeplane/files/10610610/FP_1_8_6_build_error_log.txt)

I need to use version 1.8.6 because I have many scripts and a lot of them are not compatible with newer versions, and right now I don't have time to fix these compatibility issues. 

Please help me. Thank you.

**Comment by dpolivaev** [comment, marked as answer]
You can download old versions from https://sourceforge.net/projects/freeplane/files/freeplane%20stable/archive/
But you can not easily build them because their dependencies were taken from repositories that have been shut down.

https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/
